storage/container_storage_interface/persistent-storage-csi-ember.adoc

```diff
@@ -0,0 +1,29 @@
+[id="persistent-storage-csi-ember"]
+= Ember-CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ember
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Ember-CSI.
+
+:FeatureName: Ember-CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+The Ember-CSI Operator provides a GUI to configure and deploy supported storage drivers, as well as a CSI storage class that you can use to create PVs.
+
+With each Ember-CSI back end deployed by the operator, you can create and mount Ember-CSI PVs.
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
```

Comment:
s/with a Container Storage Interface (CSI) Operator/with a CSI Operator
In this case, CSI was already defined in the first paragraph. Realizing that I've done the same thing in other assemblies. 🧇 😁